mqtt/mqtt3/examples/publisher.rs

```diff
@@ -117,7 +117,7 @@ fn main() {
     let publish_handle = client
         .publish_handle()
         .expect("couldn't get publish handle");
-    runtime_handle.clone().spawn(async move {
+    runtime.spawn(async move {
```

Comment:
Why this change?

Reply:
Clippy was upset with unnecessary cloning.